node-notes-cli/add.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+const contents = require('./data.json');
+
+const add = function () {
+  contents.notes[contents.nextId] = process.argv[3];
```

Comment:
I'd suggest (optionally) restricting direct interactions with `process` to the entry point file (`app.js`), then pass any values from it into the modules that need them.